client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -20,33 +20,29 @@ import {
 	PlanSlug,
 	isWooExpressPlusPlan,
 } from '@automattic/calypso-products';
+import {
+	JetpackLogo,
+	BloombergLogo,
+	CloudLogo,
+	CNNLogo,
+	CondenastLogo,
+	DisneyLogo,
+	FacebookLogo,
+	SalesforceLogo,
+	SlackLogo,
+	TimeLogo,
+	VIPLogo,
+	WooLogo,
+} from '@automattic/components';
 import { isHostingFlow, isLinkInBioFlow, isNewsletterFlow } from '@automattic/onboarding';
 import { MinimalRequestCartProduct } from '@automattic/shopping-cart';
 import { Button } from '@wordpress/components';
 import classNames from 'classnames';
-import {
-	localize,
-	LocalizedComponent,
-	LocalizeProps,
-	TranslateResult,
-	useTranslate,
-} from 'i18n-calypso';
+import { localize, LocalizedComponent, LocalizeProps, useTranslate } from 'i18n-calypso';
 import { Component, createRef } from 'react';
 import { connect } from 'react-redux';
-import BloombergLogo from 'calypso/assets/images/onboarding/bloomberg-logo.svg';
-import cloudLogo from 'calypso/assets/images/onboarding/cloud-logo.svg';
-import CNNLogo from 'calypso/assets/images/onboarding/cnn-logo.svg';
-import CondenastLogo from 'calypso/assets/images/onboarding/condenast-logo.svg';
-import DisneyLogo from 'calypso/assets/images/onboarding/disney-logo.svg';
-import FacebookLogo from 'calypso/assets/images/onboarding/facebook-logo.svg';
-import SalesforceLogo from 'calypso/assets/images/onboarding/salesforce-logo.svg';
-import SlackLogo from 'calypso/assets/images/onboarding/slack-logo.svg';
-import TimeLogo from 'calypso/assets/images/onboarding/time-logo.svg';
-import vipLogo from 'calypso/assets/images/onboarding/vip-logo.svg';
-import wooLogo from 'calypso/assets/images/onboarding/woo-logo.svg';
 import QueryActivePromotions from 'calypso/components/data/query-active-promotions';
 import FoldableCard from 'calypso/components/foldable-card';
-import JetpackLogo from 'calypso/components/jetpack-logo';
```

Comment:
Comparison grid probably needs updating [here](https://github.com/Automattic/wp-calypso/blob/update/move-logos-to-components/client/my-sites/plan-features-2023-grid/components/plan-comparison-grid.tsx#L23).